lesson35/js/myscript.js

```diff
@@ -0,0 +1,58 @@
+$(document).ready(function(){
```

Comment:
замість такого запису можно використовувати коротший 
```suggestion
$(function() {
```